cli/main.go

```diff
@@ -168,6 +170,9 @@ func setupSignalHandler() {
 			if fatalSignal(nativeSignal) {
 				kataLog.WithField("signal", sig).Error("received fatal signal")
 				die()
+			} else if debug && nonFatalSignal(nativeSignal) {
```

Comment:
Just wondering if `nonFatalSignal()` is appropriate naming here, when the whole thing is only about handling a specific signal to generate a backtrace.

Reply:
I chose that name as I thought we might want to extend the list of signals at some future point. As the code is trying to hide the implementation details from `main()`, what would you suggest? We could call it something like `signalIsUSR1()` but that sort of breaks the info hiding ;)

Reply:
Okay fair enough ;)